### PR TITLE
Searching

### PR DESCRIPTION
**Pull request by xDziugas**
Added an initial version of the search bar to library page.
image

### CODE REVIEW

**Comment by xDziugas** [comment]
*Currently searching only works for quiz titles

**Comment by MrMnt** [comment]
> *Currently searching only works for quiz titles

Do we need it to work for something else as well? I thought that was the plan :)

File: Services/Library/LibraryService.cs [hunk 1]
```diff
@@ -1,8 +1,10 @@
 ﻿using Acuedify.Data;
 using Acuedify.Models;
 using Acuedify.Services.Library.Interfaces;
+using Acuedify.Services.Playing.Interfaces;
 using Microsoft.EntityFrameworkCore;
 using Microsoft.IdentityModel.Tokens;
+using System.Text.RegularExpressions;
```
Comment (MrMnt) [inline Services/Library/LibraryService.cs]: I assume these using statements are not needed as there are no other changes in file?

Comment (xDziugas) [reply Services/Library/LibraryService.cs]: no, forgot to remove when I decided to move the functions needed elsewhere

**Comment by xDziugas** [comment]
> > *Currently searching only works for quiz titles
> 
> Do we need it to work for something else as well? I thought that was the plan :)

I wanted to highlight this, since I could also include quiz descriptions to the search if needed.

**Review by MrMnt — APPROVED** [review]

**Pull request merged**